html/demos/streaming.js

```diff
@@ -248,7 +248,16 @@ $(document).ready(function() {
 										}
 									});
 									Janus.attachMediaStream($('#remotevideo' + mid).get(0), stream);
-									$('#remotevideo' + mid).get(0).play();
+									var playPromise = $('#remotevideo' + mid).get(0).play();
+									if (playPromise !== undefined) {
+										playPromise
+											.then(_ => {
+												console.log('start playing')
+											})
+											.catch(error => {
+												console.log('failed to play', error)
```

Comment:
replace `console.log` with `Janus.error` and start the error message with an uppercase letter.

Reply:
Also we use the format 
```
.then(function() { 
    ...
```